<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>502</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>502</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="MainWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>502</width>
      <height>350</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="serverTab">
     <attribute name="title">
      <string>Serwer</string>
     </attribute>
     <widget class="QPushButton" name="startserverButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>219</y>
        <width>480</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stopserverButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>269</y>
        <width>480</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QTextBrowser" name="serverstatusTextBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>480</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;Serwer jest obecnie wyłączony.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="serveractionsTextBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>480</width>
        <height>160</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="clientTab">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <attribute name="title">
      <string>Klient</string>
     </attribute>
     <widget class="QPushButton" name="bokconnectButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>72</y>
        <width>291</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Połącz z obsługą klienta</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bokdisconnectButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>72</y>
        <width>181</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Rozłącz</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="clientnameEdit">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>311</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="clientnameLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>13</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Wpisz imię i nazwisko:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="clientproblemBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>311</width>
        <height>27</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Panie... kompjuter mi nie działa...</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Olaboga... nie ma internetów...</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mam taki problem, że drukarka nie drukuje...</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Zapomniałem hasło do księgi twarzy...</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mam jakiś problem, ale nie wiem jaki...</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="clientproblemLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>45</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Wybierz typ problemu:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="clientTextEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>370</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="clientsendButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>280</y>
        <width>100</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Wyślij</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="clientmessageTextEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>480</width>
        <height>160</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>480</width>
        <height>160</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>480</width>
        <height>160</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="consultantTab">
     <attribute name="title">
      <string>Konsultant</string>
     </attribute>
     <widget class="QPushButton" name="serverconnectButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>81</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Połącz</string>
      </property>
     </widget>
     <widget class="QPushButton" name="serverdisconnectButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>10</y>
        <width>81</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Rozłącz</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="consultantnameEdit">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>211</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="consultantnameLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>14</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Twoje imię:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="consultantclientchooseBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>301</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="consultantchooseButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>40</y>
        <width>81</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Wybierz</string>
      </property>
     </widget>
     <widget class="QPushButton" name="consultantrefreshButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>40</y>
        <width>81</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Odśwież</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="consultantmessageTextEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>480</width>
        <height>160</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
     <widget class="QPushButton" name="consultantsendButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>280</y>
        <width>100</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Wyślij</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="consultantTextBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>370</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="consultantclientdataTextBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>74</y>
        <width>480</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
